docs/components/layout/Documentation/helpers/CodeBlock.js

```
@@ -39,7 +42,23 @@ const CodeBlock = ({ children, language = 'jsx' }) => {
     
     return (
         <pre className="relative mb-8">
-            <code className={`language-${language} whitespace-pre-wrap`} style={{ wordBreak: 'break-word' }}>{code}</code>
+           <div className="relative ">
+           <code className={`language-${language} whitespace-pre-wrap`} style={{ wordBreak: 'break-word' }}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)